From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: GerryYuu <jjjooo233@163.com>
Date: Sun, 19 Jun 2022 12:42:18 +0800
Subject: [PATCH] Setup project


diff --git a/build.gradle.kts b/build.gradle.kts
index a4a146d9c3e07bdf941d81147ca31daf4689a687..8afe6f5905fc9a4497f3b604d9f83f959129b307 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -9,7 +9,7 @@ plugins {
 }
 
 dependencies {
-    implementation(project(":pufferfish-api")) // Pufferfish // Paper
+    implementation(project(":bubble-api")) // Bubble // Pufferfish // Paper
     // Pufferfish start
     implementation("io.papermc.paper:paper-mojangapi:1.19-R0.1-SNAPSHOT") {
         exclude("io.papermc.paper", "paper-api")
